stacks/analytics-ingest-lambda.yml

```diff
@@ -149,6 +175,45 @@ Resources:
       VpcConfig:
         SecurityGroupIds: !Split [",", !FindInMap [VPCSettingsMap, !Ref EnvironmentTypeAbbreviation, vpcsecurity]]
         SubnetIds: !Split [",", !FindInMap [VPCSettingsMap, !Ref EnvironmentTypeAbbreviation, vpcsubnets]]
+  AnalyticsDynamodbLambdaFunction:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Code:
+        S3Bucket: !Ref CodeS3Bucket
+        S3Key: lambda/PRX-analytics-ingest-lambda.zip
+        S3ObjectVersion: !Ref CodeS3ObjectVersion
+      Description: Dovetail analytics to dynamodb
+      Environment:
+        Variables:
+          DYNAMODB: "true"
```

Comment:
This is fairly opaque/ambiguous variable name

Reply:
Maybe just a comment to describe what it does for now

Reply:
Yeah, the way the lambda code decides which of the 4 "modes" to run in is keyed to these kind of odd variables.  I'll update them to some sort of `MODE: "ddb"` / `MODE: "redis"` at some point.  Will open a ticket - requires some coordination between CFN/lambda-code.

Reply:
PRX/analytics-ingest-lambda#34